common/presenters/framework-field-summary-list-row.test.js

```diff
@@ -611,7 +611,7 @@ describe('Presenters', function () {
               {
                 value: test.value,
                 valueType: test.valueType,
-                responded: false,
+                responded: true,
```

Comment:
Do you need a test for the `false` value too?

Reply:
In this instance it should never actually be `false`. That was the issue. When this presenter is called, for the add another fields it should always set responded to `true` so that the value gets displayed.